### ISSUE TITLE
Bug 58071760: [React Native Gallery -> TrackPlayer]: Narrator fails to announce the success information on activating 'Next' or 'Previous' button.

### ISSUE THREAD

**Issue by vineethkuttan** [microsoft/react-native-gallery#615]
Build Details:
OS Version: Dev (OS Build 27871.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.79.0
AT: Screen Reader
 
Pre-Requisite:
Turn on Narrator (Ctrl + Win + Enter).

Repro-Steps:
Launch the React Native Gallery app.
Navigate to ‘Media' button present in left navigation pane and expand it.
Navigate to 'TrackPlayer' button and activate it.
Navigate to 'Next' and 'Previous' button and activate it.
Listen to narrator announcement.

Actual Result:

Narrator fails to announce the success information on activating 'Next' or 'Previous' button. When user activates the button narrator stays on mute.

Expected Result:
Screen reader should announce the success information on activating 'Next' or 'Previous' button. When user activates 'Next' button narrator should announce as "XYZ track player, next button".

User Impact:
Users with visual impairments who rely on screen reader will be impacted and user will not be able to whether the performed action is completed or not.

MAS Reference:
[MAS 4.2.12 – Change Notification](https://aka.ms/MAS4.2.12)

Trap ID:
[1.9 - Feedback Failure](https://aka.ms/FeedbackFailure)

"Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”

**Comment by chiaramooney** [comment]
This issue will be a little more tricky to tackle because it will involve editing the react-native-track-player module. Luckily the change should only be a JS side change (no Paper architecture specific knowledge needed).

Note: The source code for the Paper React Native Gallery begins at the root of the repo rather than inside the New Arch folder.

**Comment by vineethkuttan** [comment]
This one use react-native-track-player , where for buttons we didn't use announceForAccessibility.

This module is not supported in Fabric as for now. And it is not part of the Fabric gallery.

**Comment by chiaramooney** [comment]
> This one use react-native-track-player , where for buttons we didn't use announceForAccessibility.
> 
> This module is not supported in Fabric as for now. And it is not part of the Fabric gallery.

Will need to be fixed for Paper otherwise Paper will lose it's C (Compliant) accessibility grading.
